swagger2.cabal

```diff
@@ -96,6 +98,9 @@ test-suite spec
                   , unordered-containers
                   , vector
                   , lens
+  if !impl(ghc >= 8.0)
+     build-depends:    semigroups >= 0.18.3 && <0.19
```

Comment:
no need to specify bounds in test-suite (as they are inherited from library). This helps if/when `semigroups-0.19` is released.